harness/testTypedArray.js

```diff
@@ -29,8 +30,17 @@ var typedArrayConstructors = [
   Uint8ClampedArray
 ];
 
-var floatArrayConstructors = typedArrayConstructors.slice(0, 2);
-var intArrayConstructors = typedArrayConstructors.slice(2, 7);
```

Comment:
`intArrayConstructors` did not contain `Uint8ClampedArray` but it is included now. This seems to be an intended change. Unfortunately, some `Atomics` tests (like `built-ins/Atomics/add/good-views.js`) still use `intArrayConstructors`. In other words, these tests are incorrect now. They should be rewritten to use `testWithAtomicsFriendlyTypedArrayConstructors()`, I guess.

Reply:
Oh, that was not an intended change, I just didn't realize that the `.slice` above stopped before the end of the list.

In fact the `.slice` was previously also omitting `Uint8Array`, which seems wrong. I've left it in, but opened https://github.com/tc39/test262/pull/3984 to remove Uint8ClampedArray.

Reply:
I think that the original idea was to include `Uint8ClampedArray` in `intArrayConstructors`. The author just haven't realized that the second argument in `typedArrayConstructors.slice(2, 7);` is not the length but an (excluded) end index. That's why `Uint8Array` and `Uint8ClampedArray` were missing by accident. I find it correct to include `Uint8ClampedArray` in this array (it is a typed array with int elements). Some tests may skip `Uint8ClampedArray` unnecessarily when it is not included. That's why I though that the change was intentional and that just the problematic Atomics tests (that use `intArrayConstructors` without excluding `Uint8ClampedArray`) should be fixed/updated.

Reply:
The following tests started to fail (when run against `graal-js` engine). So, this is the correct list, hopefully.
```
Unexpectedly failed 'built-ins/Atomics/add/bad-range.js'
Unexpectedly failed 'built-ins/Atomics/add/good-views.js'
Unexpectedly failed 'built-ins/Atomics/and/bad-range.js'
Unexpectedly failed 'built-ins/Atomics/and/good-views.js'
Unexpectedly failed 'built-ins/Atomics/compareExchange/bad-range.js'
Unexpectedly failed 'built-ins/Atomics/compareExchange/good-views.js'
Unexpectedly failed 'built-ins/Atomics/exchange/bad-range.js'
Unexpectedly failed 'built-ins/Atomics/exchange/good-views.js'
Unexpectedly failed 'built-ins/Atomics/load/bad-range.js'
Unexpectedly failed 'built-ins/Atomics/load/good-views.js'
Unexpectedly failed 'built-ins/Atomics/or/bad-range.js'
Unexpectedly failed 'built-ins/Atomics/or/good-views.js'
Unexpectedly failed 'built-ins/Atomics/store/bad-range.js'
Unexpectedly failed 'built-ins/Atomics/store/good-views.js'
Unexpectedly failed 'built-ins/Atomics/sub/bad-range.js'
Unexpectedly failed 'built-ins/Atomics/sub/good-views.js'
Unexpectedly failed 'built-ins/Atomics/xor/bad-range.js'
Unexpectedly failed 'built-ins/Atomics/xor/good-views.js'
```

Reply:
@iamstolis Thanks, see https://github.com/tc39/test262/pull/3984.

Reply:
Thanks, I run the tests from the current revision (`62294b592e332bc16da6114e461fd7a8ed90daab`) of https://github.com/tc39/test262/pull/3984 against `graal-js` and I haven't seen any unexpected failures.